ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -29,22 +29,29 @@
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-#include <aws/core/Aws.h>
 #include "s3_connection.h"
+#include "s3_log_system.h"
+#include <aws/core/Aws.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/logging/DefaultLogSystem.h>
```

Comment:
Pretty sure we don't need this include. Will you check if the ones above and below are needed as well?

Reply:
Yep! Have removed the unused ones.